### DISCUSSION TITLE
Method Chaining

### DISCUSSION THREAD

**Discussion by bubbajoe** [asticode/go-astiav#76] [Ideas]
Hello again! I was wondering if you would be open to using the method chaining design pattern for the Setters so that changing values can look much better.

Usage example:
```
cp := astiav.AllocCodecParameters().
                  SetHeight(1).
                  SetHeight(2).
                  SetSampleRate(3)
```

**Comment by asticode** [comment]
Hey, I'm not looking for this change right now, but I'll keep the issue open in case that changes
